File: BookshopClient/src/app/shared/models/wishList.ts
import { ICartItem } from "./shoppingCart";
import { v4 as uuidv4 } from "uuid";

export interface IWishList {
    id: string;
    items: ICartItem[];
}

export class WishList implements IWishList {
    id = uuidv4();
    items: ICartItem[] = [];
}

File: BookshopServer/Entities/WishList.cs
namespace BookshopServer.Entities
{
    public class WishList
    {
        public WishList()
        {
                
        }

        public WishList(string id)
        {
            Id = id;
        }

        public string Id { get; set; }
        public List<CartItem> Items { get; set; } = new List<CartItem>();
    }
}
